tests/osquery/test_setup_distributed_queries.py

```diff
@@ -330,6 +330,61 @@ def test_distributed_query_machines(self, get_paginate_by):
         self.assertContains(response, "Error")
         self.assertContains(response, err_msgs[-1])
 
+    # distributed query search
+
+    def test_distributed_query_machines_serial_search(self):
+        distributed_query = self._force_distributed_query()
+        dqm_count = 3
+        serial_numbers = [get_random_string(12) for _ in range(dqm_count)]
+        serial_search = serial_numbers[0]
+        err_msgs = [f"Error Message {_}" if _ > 0 else "-" for _ in range(dqm_count)]
```

Comment:
No need to test the error messages in this test. Keep it simple! They can be left as `None` with `status = 0`. Also, when a variable is used, it is better to not name it `_`.